source/git-util.js

```diff
@@ -128,9 +128,9 @@ const gitVersion = async () => {
 
 exports.verifyRecentGitVersion = async () => {
 	const installedVersion = await gitVersion();
-	const minVersion = '2.11.0';
+	const requiredGitVersion = require('../package.json').engines.git;
 
-	if (!version(minVersion).isGreaterThanOrEqualTo(installedVersion)) {
-		throw new Error('Please upgrade to git@2.11.0 or newer.');
```

Comment:
The `... or newer` shouldn't be removed (here as well as in the other respective places).

Reply:
requiredGitVersion is ">=x.y.z", so the "... or newer" is no longer needed.

Reply:
I forgot about that 😅 One option then is to change the format to

```js
`Please upgrade to git${requiredGitVersion}` //=> Please upgrade to git>=2.11.0
```

However, I'd prefer it if we could keep the old format by stripping the comparison operators from the range to extract the minimum version (we don't need to worry about maximum versions at the moment).